js/src/components/account-card/index.js

```diff
@@ -38,11 +40,24 @@ const googleLogo = (
 	/>
 );
 
+const wpLogo = (
+	<img
+		src={ wpLogoURL }
+		alt={ __( 'WordPress.com Logo', 'google-listings-and-ads' ) }
+		width="40"
+		height="40"
+	/>
+);
+
 const appearanceDict = {
 	[ APPEARANCE.EMPTY ]: {},
+	[ APPEARANCE.WPCOM ]: {
+		icon: wpLogo,
+		title: 'WordPress.com',
+	},
 	[ APPEARANCE.GOOGLE ]: {
 		icon: googleLogo,
-		title: __( 'Google account', 'google-listings-and-ads' ),
+		title: __( 'Google', 'google-listings-and-ads' ),
```

Comment:
📝 
I checked several recent designs on Figma. All Google account cards no longer have "account" in the title, so I removed it here.